examples/gcoap/gcoap_cli.c

```diff
@@ -153,6 +153,16 @@ int gcoap_cli_cmd(int argc, char **argv)
                 if (!_send(&buf[0], len, argv[2], argv[3])) {
                     puts("gcoap_cli: msg send failed");
                 }
+                else {
+                    /* send Observe notification for /cli/stats */
+                    int res = gcoap_obs_init(&pdu, &buf[0], GCOAP_PDU_BUF_SIZE,
+                                                            &_resources[0]);
+                    if (res == 0) {
```

Comment:
You can save an int and give more debug info with the following:
```c
switch (gcoap_obs_init(&pdu, &buf[0], GCOAP_PDU_BUF_SIZE, &_resources[0]) {
case GCOAP_OBS_INIT_OK:
    DEBUG("[info] gcoap observer initialized with success\n");
    [...] /* your obs send code */
    break;
case GCOAP_OBS_INIT_ERR:
    DEBUG("[error] cannot initialize gcoap observer\n");
    break;
case GCOAP_OBS_INIT_NOOBS:
    DEBUG("[error]  no observer for resource '%s'\n", &_resources[0]);
    break;
}
```

or something similar